R/cnd-abort.R

```diff
@@ -1364,6 +1365,30 @@ call_restore <- function(x, to) {
 #' # stop("foo")
 NULL
 
+#' Display backtrace on warning
+#'
+#' @description
+#' Similarly to [`rlang_backtrace_on_error_report`], you can set
```

Comment:
Should these actually be documented in the same file?

Reply:
You're right, I merged this inside the Rmd section of backtrace-on-error.